Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,84 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+            //Set variable
+            bool backtotop = true;
+            string option;
```

Comment:
option can be in the scope of the while loop - initialize directly

Reply:
Moved option to Whilte loop